tests/ragger/app/command_builder.py

```diff
@@ -109,27 +116,28 @@ def eip712_send_struct_impl_struct_field(self, data: bytearray) -> Iterator[byte
                                   data_w_length[:0xff])
             data_w_length = data_w_length[0xff:]
 
-    def _format_bip32(self, bip32, data: bytearray) -> bytearray:
-        data.append(len(bip32))
-        for idx in bip32:
-            data.append((idx & 0xff000000) >> 24)
-            data.append((idx & 0x00ff0000) >> 16)
-            data.append((idx & 0x0000ff00) >> 8)
-            data.append((idx & 0x000000ff))
+    def _format_bip32(self, path: list[Optional[int]], data: bytearray) -> bytearray:
+        data.append(len(path))
+        for p in path:
+            if p == None:
+                val = 0
+            else:
+                val = (0x8 << 28) | p
+            data += struct.pack(">I", val)
         return data
```

Comment:
Use the functions from `ragger.bip`, it does the same thing + handles hardening !

Reply:
Did not know Ragger had this, it's now using it. :+1: